app/services/permission_service.rb

```diff
@@ -28,6 +28,7 @@ def allow?(controller, action)
       return true if controller == 'locations'        && action.in?(%w(index show))
       return true if controller == 'stores/cars'      && action.in?(%w(index show new create update))
       return true if controller == 'stores/dashboard' && action == 'show'
+      return true if controller == 'users'            && action == 'profile'
```

Comment:
Operator `&&` should be surrounded with a single space.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.